jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationTemplateIndex.kt

```diff
@@ -54,8 +54,9 @@ class CloudFormationTemplateIndex : FileBasedIndexExtension<String, MutableList<
         val indexedResources = mutableMapOf<String, MutableList<IndexedResource>>()
 
         fileContent.psiFile.acceptNode(object : PsiElementVisitor() {
-            override fun visitElement(element: PsiElement?) {
+            override fun visitElement(element: PsiElement) {
                 super.visitElement(element)
+                // element is nullable in versions prior to 2020.1 FIX_WHEN_MIN_IS_201
```

Comment:
what's the fix here?

Reply:
making it not nullable and leaving in `?.`. It seems like it wasn't actually nullable before but the `@NotNull` assertion was just added, so at this point I'm not sure what we should do here